primitives/src/merkle_tree/field_based_mht/parameters/tweedle_fq.rs

```diff
@@ -0,0 +1,79 @@
+use algebra::{
+    fields::tweedle::Fq as TweedleFq,
+    biginteger::BigInteger256,
+    field_new,
+};
+use crate::{
+    crh::TweedleFqPoseidonHash,
+    merkle_tree::field_based_mht::FieldBasedMerkleTreePrecomputedEmptyConstants
+};
+
+pub const TWEEDLE_MHT_POSEIDON_PARAMETERS: FieldBasedMerkleTreePrecomputedEmptyConstants<'static, TweedleFqPoseidonHash> =
+    FieldBasedMerkleTreePrecomputedEmptyConstants {
```

Comment:
These constants need to be recomputed when using the new Poseidon parameters from [UpdatedPoseidonParameters](https://github.com/HorizenOfficial/ginger-lib/tree/UpdatedPoseidonParameters).